packages/bits/src/lib/toolbar/toolbar-keyboard.service.ts

```diff
@@ -2,6 +2,11 @@ import { Injectable } from "@angular/core";
 import { KEYBOARD_CODE } from "../../constants";
 import { MenuComponent } from "../menu";
 
+enum NavDirection {
```

Comment:
Thanks for adding the enum. Do you mind exporting it from [this](https://github.com/solarwinds/nova/blob/f1dbe170b14cc999ce5233b19844a093d43fc66e/packages/bits/src/lib/toolbar/public-api.ts) file and using the enum instead of a number in all the places where it can be used? For example in the service spec:

```
service["navigateFromMoreBtn"](NavDirection.Left, buttonsMock)
```

Reply:
And, one last thing, do you mind making the entries of the enum PascalCase? We've been trying to go with that for a convention. I'll see if it's documented somewhere. If not, I'll add it to the style guide. Thanks in advance.

e.g. `Left` instead of `LEFT`

Reply:
remove it at all.